lockbox-ios/Common/AppDelegate.swift

```diff
@@ -17,6 +19,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         self.window?.rootViewController = RootView()
         self.window?.makeKeyAndVisible()
 
+        if UserDefaults.standard.string(forKey: FirstRunKey) == nil {
```

Comment:
I stumbled a little over this.  The flag is **not** set _until_ first run completes. 

I can't think of a better naming convention here, but it might be worth a comment for future us.